（docs）增加英语翻译，（fix）修复数个错误

Kimimaybe29 opened the pull request.

Kimimaybe29 committed:
加上了头像的webp

Kimimaybe29 committed:
加上了头像的webp

Kimimaybe29 committed:
Revert "加上了头像的webp"

This reverts commit f1942964dad82a7da44fd23c64ec3765234545c9.

Kimimaybe29 committed:
Revert "加上了头像的webp"

This reverts commit f1942964dad82a7da44fd23c64ec3765234545c9.

LorienYang committed:
update(docs):更新内容编写准则、Bvideo组件增加手机适配、部分细节修复

Kimimaybe29 committed:
Merge branch 'Linking-ET:main' into main

Kimimaybe29 committed:
完成主项目kimi分支的所有更改以及优化了部分界面的排版格式

涉及commit：
https://github.com/Linking-ET/linking-et.github.io/commit/6377fe517280dbebcaa55c558071e0b0f4038568
至
https://github.com/Linking-ET/linking-et.github.io/commit/cfe84e990647abb5c2bdbb52cd5d70869af5b57f

Kimimaybe29 committed:
完成主项目kimi分支的所有更改以及优化了部分界面的排版格式

涉及commit：
https://github.com/Linking-ET/linking-et.github.io/commit/6377fe517280dbebcaa55c558071e0b0f4038568
至
https://github.com/Linking-ET/linking-et.github.io/commit/cfe84e990647abb5c2bdbb52cd5d70869af5b57f

XChen446 committed:
Remove comment about team members from team.md

Removed a comment about team members from the document.

Kimimaybe29 committed:
继续深化排版

Kimimaybe29 committed:
Merge branch 'develop' into main

Kimimaybe29 committed:
Merge pull request #53 from Kimimaybe29/main

排版改完了

XChen446 committed:
Clear lead template in team.md

Removed the text '除了xc都是懒狗' from the lead template.

XChen446 committed:
Update server access information in documentation

Kimimaybe29 committed:
更新英语首页

LorienYang committed:
remove(docs):删除残余图片

LorienYang committed:
fix(docs):修正图片引用

LorienYang committed:
fix(docs):再次修正图片引用

XChen446 committed:
Add 'develop' branch to pull request checks

Kimimaybe29 committed:
增加英文页脚

Kimimaybe29 committed:
增加英文页脚

LorienYang committed:
add(docs):增加英语页脚

LorienYang committed:
add(docs):增加英语页脚

Kimimaybe29 committed:
Custom Texts, nav, sidebar翻译完毕

引入文档

Kimimaybe29 committed:
Custom Texts, nav, sidebar翻译完毕

引入文档

Kimimaybe29 committed:
英语页面稍后翻译，现在这样应该能合了 (#56)

* 增加英文页脚

* add(docs):增加英语页脚

* Custom Texts, nav, sidebar翻译完毕

引入文档

---------

Co-authored-by: LorienYang <yangluoying19277904322@outlook.com>

Kimimaybe29 committed:
Revert "英语页面稍后翻译，现在这样应该能合了 (#56)" (#57)

This reverts commit b8f03a1e04f6b0b3796e24369eb7d9ce37a830bc.

Kimimaybe29 committed:
(docs)完成guide目录下所有可访问md文件的翻译(ai帮翻)(fix)一些错误得到修正

Kimimaybe29 committed:
(docs)完成guide目录下所有可访问md文件的翻译(ai帮翻)(fix)一些错误得到修正

Kimimaybe29 committed:
(doc)翻译完毕about下所有文档，(fix)修复数个小错误

Kimimaybe29 committed:
（docs）完成各个服务器（除了水服）的翻译，还有叶服怎么是空的？

Kimimaybe29 committed:
(docs)翻译完毕rule下所有文档（fix）修复数个错误

Kimimaybe29 committed:
Merge branch 'develop' into in_dev_kimi_do_not_interrupt

Kimimaybe29 committed:
（fix）修复一个错误，workflow应当已没有问题

Kimimaybe29 committed:
（Fix）彻底修复

Corrected the link format for better navigation in the documentation.

LorienYang committed:
fix(docs):修复Kimi留下的历史遗留问题(憋笑

Kimimaybe29 committed:
(docs)翻译完毕所有英语页面，(fix)修复数个中文页面中的小错误并同步英语页面 (#61)

* 增加英文页脚

* add(docs):增加英语页脚

* Custom Texts, nav, sidebar翻译完毕

引入文档

* (docs)完成guide目录下所有可访问md文件的翻译(ai帮翻)(fix)一些错误得到修正

* (doc)翻译完毕about下所有文档，(fix)修复数个小错误

* （docs）完成各个服务器（除了水服）的翻译，还有叶服怎么是空的？

* (docs)翻译完毕rule下所有文档（fix）修复数个错误

* （fix）修复一个错误，workflow应当已没有问题

* （Fix）彻底修复

Corrected the link format for better navigation in the documentation.

* fix(docs):修复Kimi留下的历史遗留问题(憋笑

---------

Co-authored-by: LorienYang <yangluoying19277904322@outlook.com>

Kimimaybe29 committed:
Merge branch 'main' into develop

XChen446 committed:
Update Bedrock Entry status message

XChen446 committed:
Add warning for offline account setup location

添加了对非大陆地区使用离线账户的警告

XChen446 committed:
recommend using the version matching

XChen446 committed:
咬文嚼字ing

XChen446 committed:
Delete docs/en/docs/guide/howtopr.md

XChen446 commented on docs/en/docs/rule/MsgRule.md:
我不好评价，国外佬可以直接去dc更适用其实（没必要这里还写qq群限制吧

Kimimaybe29 replied:
那我直接删掉这一页？不想翻了，这段就不适合给老外看

XChen446 replied:
都可以

Kimimaybe29 replied:
已删除，是否同意更新网站，就等你了

Kimimaybe29 committed:
(fix)删除英语页面中的安全发言注意事项，对于英语用户不适合

Kimimaybe29 committed:
Revert "(fix)删除英语页面中的安全发言注意事项，对于英语用户不适合"

This reverts commit 856777658ddad0df187e3feb60c9e6195abe8837.

Kimimaybe29 committed:
Merge branch 'in_dev_kimi_do_not_interrupt' into develop

Kimimaybe29 committed:
(fix)根据XCHEN446的建议，删除英语文本中安全发言的内容 (#65)

* 同步develop的进度到in_dev (#64)

* 完成主项目kimi分支的所有更改以及优化了部分界面的排版格式 (#51)

* 加上了头像的webp

* Revert "加上了头像的webp"

This reverts commit f1942964dad82a7da44fd23c64ec3765234545c9.

* update(docs):更新内容编写准则、Bvideo组件增加手机适配、部分细节修复

* 完成主项目kimi分支的所有更改以及优化了部分界面的排版格式

涉及commit：
https://github.com/Linking-ET/linking-et.github.io/commit/6377fe517280dbebcaa55c558071e0b0f4038568
至
https://github.com/Linking-ET/linking-et.github.io/commit/cfe84e990647abb5c2bdbb52cd5d70869af5b57f

* 加上了头像的webp

* Revert "加上了头像的webp"

This reverts commit f1942964dad82a7da44fd23c64ec3765234545c9.

* 完成主项目kimi分支的所有更改以及优化了部分界面的排版格式

涉及commit：
https://github.com/Linking-ET/linking-et.github.io/commit/6377fe517280dbebcaa55c558071e0b0f4038568
至
https://github.com/Linking-ET/linking-et.github.io/commit/cfe84e990647abb5c2bdbb52cd5d70869af5b57f

* Remove comment about team members from team.md

Removed a comment about team members from the document.

* 继续深化排版

* Clear lead template in team.md

Removed the text '除了xc都是懒狗' from the lead template.

* Update server access information in documentation

---------

Co-authored-by: LorienYang <yangluoying19277904322@outlook.com>
Co-authored-by: XChen446 <3088576338@qq.com>

* remove(docs):删除残余图片 (#54)

* 加上了头像的webp

* Revert "加上了头像的webp"

This reverts commit f1942964dad82a7da44fd23c64ec3765234545c9.

* update(docs):更新内容编写准则、Bvideo组件增加手机适配、部分细节修复

* 完成主项目kimi分支的所有更改以及优化了部分界面的排版格式

涉及commit：
https://github.com/Linking-ET/linking-et.github.io/commit/6377fe517280dbebcaa55c558071e0b0f4038568
至
https://github.com/Linking-ET/linking-et.github.io/commit/cfe84e990647abb5c2bdbb52cd5d70869af5b57f

* 加上了头像的webp

* Revert "加上了头像的webp"

This reverts commit f1942964dad82a7da44fd23c64ec3765234545c9.

* 完成主项目kimi分支的所有更改以及优化了部分界面的排版格式

涉及commit：
https://github.com/Linking-ET/linking-et.github.io/commit/6377fe517280dbebcaa55c558071e0b0f4038568
至
https://github.com/Linking-ET/linking-et.github.io/commit/cfe84e990647abb5c2bdbb52cd5d70869af5b57f

* Remove comment about team members from team.md

Removed a comment about team members from the document.

* 继续深化排版

* Clear lead template in team.md

Removed the text '除了xc都是懒狗' from the lead template.

* Update server access information in documentation

* 更新英语首页

* remove(docs):删除残余图片

* fix(docs):修正图片引用

* fix(docs):再次修正图片引用

* Add 'develop' branch to pull request checks

---------

Co-authored-by: Kimimaybe29 <Kimishishengye@outlook.com>
Co-authored-by: XChen446 <3088576338@qq.com>

* (docs)翻译完毕所有英语页面，(fix)修复数个中文页面中的小错误并同步英语页面 (#61)

* 增加英文页脚

* add(docs):增加英语页脚

* Custom Texts, nav, sidebar翻译完毕

引入文档

* (docs)完成guide目录下所有可访问md文件的翻译(ai帮翻)(fix)一些错误得到修正

* (doc)翻译完毕about下所有文档，(fix)修复数个小错误

* （docs）完成各个服务器（除了水服）的翻译，还有叶服怎么是空的？

* (docs)翻译完毕rule下所有文档（fix）修复数个错误

* （fix）修复一个错误，workflow应当已没有问题

* （Fix）彻底修复

Corrected the link format for better navigation in the documentation.

* fix(docs):修复Kimi留下的历史遗留问题(憋笑

---------

Co-authored-by: LorienYang <yangluoying19277904322@outlook.com>

* Update Bedrock Entry status message

* Add warning for offline account setup location

添加了对非大陆地区使用离线账户的警告

* recommend using the version matching

* 咬文嚼字ing

* Delete docs/en/docs/guide/howtopr.md

---------

Co-authored-by: LorienYang <yangluoying19277904322@outlook.com>
Co-authored-by: XChen446 <3088576338@qq.com>
Co-authored-by: Lorien Yang <80152431+LorienYang@users.noreply.github.com>

* (fix)删除英语中安全发言的内容

---------

Co-authored-by: LorienYang <yangluoying19277904322@outlook.com>
Co-authored-by: XChen446 <3088576338@qq.com>
Co-authored-by: Lorien Yang <80152431+LorienYang@users.noreply.github.com>
